src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -707,12 +730,236 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
        displss2d(i)=displss2d(i-1)+ ijns2d(i-1)
     enddo
 
+
+
+
+      it=ntguessig
+
+      
+      allocate( name_metvars2d(GSI_MetGuess_Bundle(it)%n2d))
+      allocate( name_metvars3d(GSI_MetGuess_Bundle(it)%n3d))
+      call gsi_bundleinquire (GSI_MetGuess_Bundle(it),'shortnames::2d', name_metvars2d,istatus)
+      call gsi_bundleinquire (GSI_MetGuess_Bundle(it),'shortnames::3d', name_metvars3d,istatus)
+      do i=1,GSI_MetGuess_Bundle(it)%n2d
+        write(6,*)'metvardeb333-2d nmae ', trim(name_metvars2d(i))
+      enddo
+      do i=1,GSI_MetGuess_Bundle(it)%n3d
+        write(6,*)'metvardeb333-3d name ', trim(name_metvars3d(i))
+      enddo
```

Comment:
Too many output of metvardeb333* information.  Suggest only writing out this information in pe0.

Reply:
changed as suggested